src/integration-test/java/developerTC/TestAgentEndpointQuery_06.java

```diff
@@ -1,77 +0,0 @@
-package developerTC;
-
-import org.junit.jupiter.api.Test;
-import sg.dex.starfish.Resolver;
-import sg.dex.starfish.impl.memory.LocalResolverImpl;
-import sg.dex.starfish.impl.remote.RemoteAgent;
-import sg.dex.starfish.util.DID;
-import sg.dex.starfish.util.JSON;
-import sg.dex.starfish.util.Utils;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-
-/**
- * As a developer working with Ocean,
- * I need to obtain service endpoints for an arbitrary Agent in the Ocean ecosystem
- */
-public class TestAgentEndpointQuery_06 {
-
-
-    @Test
-    public void testServiceEndPoint() {
```

Comment:
A lot of these changes / deletions see unrelated to the PR title (update for default provenance).

Should these perhaps be a separate PR?